content/docs/use-cases/shared-development-server.md

```diff
@@ -3,102 +3,110 @@
 Some teams may prefer using one single shared machine to run their experiments.
 This allows better resource utilization, such as the ability to use multiple
 GPUs, centralized data storage, etc. With DVC, you can easily setup shared data
-storage on a server accessed by several users, in a way that enables almost
-instantaneous <abbr>workspace</abbr> restoration/switching speed for everyone –
-similar to `git checkout` for your code.
+storage on a server accessed by several users or for any other reason, in a way
+that enables almost instantaneous <abbr>workspace</abbr> restoration/switching
+speed for everyone – similar to `git checkout` for your code.
 
 ![](/img/shared-server.png)
 
 ## Preparation
 
-Create a shared directory to be used as the <abbr>cache</abbr> location for
-everyone's <abbr>DVC projects</abbr>, so that all your colleagues can use the
-same project cache:
+Create a directory external to your <abbr>DVC projects</abbr> to be used as a
+shared <abbr>cache</abbr> location for everyone's projects:
 
 ```dvc
-$ mkdir -p /path/to/dvc-cache
+$ mkdir -p /home/shared/dvc-cache
 ```
 
-You will have to make sure that the directory has proper permissions setup, so
-that all your colleagues can read and write to it, and can access cache files
-written by others. The most straightforward way to do this is to make sure that
-everyone's users are members of the same group, and that your shared cache
-directory is owned by this group, with the aforementioned permissions.
+> The `/home/shared` directory used as example above is typical in Linux
```

Comment:
no, let's not include notes like this :)

Reply:
Ok, reverting. But I just copied it from https://dvc.org/doc/user-guide/managing-external-data#examples where it's already being used. Should I remove that one too?

Reply:
I'll just remove it from there too.

Reply:
👍